templates/accounts/display_user_selectable.html

```diff
@@ -0,0 +1,10 @@
+<div class="bw-selectable-object between {% if selected %}selected{% endif %}">
+    <div class="middle spacing-2">
+        <label>
+        <input type="checkbox" class="bw-checkbox" data-object-id="{{ user.id }}" {% if selected %}checked{% endif %}>
+        </label>
```

Comment:
can you select this input by clicking on the display_user? If not, then maybe we don't need the label